lib/chef_apply/action/install_chef/base.rb

```diff
@@ -116,6 +107,31 @@ def upload_to_target(local_path)
       remote_path
     end
 
+    def check_minimum_chef_version!(target)
```

Comment:
![huge fan](https://user-images.githubusercontent.com/517302/43270822-f79d2a48-90c3-11e8-9edd-e178d601a2fd.gif)

I'm a huge fan of this extracting the version checking logic to a well-named method.